src/commands/CmdCalendar.cpp

```diff
@@ -606,6 +610,25 @@ std::string CmdCalendar::renderMonths (
         // colorize due tasks
         if (config.get ("calendar.details") != "none")
         {
+          config.set ("schdeuled", 0);
```

Comment:
There is a typo here in `scheduled`.

Reply:
Hi,
thanks - sorry I missed that - it is fixed now.